pom.xml

```diff
@@ -182,7 +182,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.18.1</version>
                 <configuration>
-                    <runOrder>alphabetical</runOrder>
+                    <runOrder>random</runOrder>
```

Comment:
The previous `alphabetical` was configured to avoid the test instability, but it doesn't fix the underlying problem. I set `random` so that we can find an issue earlier.